crates/bevy_ecs/src/storage/table/mod.rs

```diff
@@ -856,7 +857,9 @@ mod tests {
         let mut table = TableBuilder::with_capacity(0, columns.len())
             .add_column(components.get_info(component_id).unwrap())
             .build();
-        let entities = (0..200).map(Entity::from_raw).collect::<Vec<_>>();
+        let entities = (0..200)
+            .map(|index| Entity::from_raw(EntityRow::new(NonMaxU32::new(index).unwrap())))
```

Comment:
```suggestion
            .filter_map(Entity::fresh_from_row)
```
Since we know `0..200` will only produce valid `Entity`'s, `filter_map` is equivalent to unwrapping.

Reply:
Yeah, but this "we know" is kinda the thing tests are meant to verify. I put unwraps everywhere in tests for that reason. Even though the test isn't strictly about that, and it's trivial to verify (as of now). I don't have a strong opinion though. If consensus is different, I'm happy to change this.

Reply:
Could meet in the middle and do:
```rust
let entities = (0..200)
    .map(Entity::fresh_from_row)
    .map(Option::unwrap)
    .collect::<Vec<_>>();
```
That's more subjective tho.

Reply:
Yeah. I'm indifferent really. Whatever consensus is is fine with me.